src/IJulia/setup.jl

```diff
@@ -92,36 +93,35 @@ mimewritable(m::MIME, s::Signal) =
     mimewritable(m, s.value)
 
 # fixes ambiguity warnings
-function writemime(io:: IO, m::MIME"text/plain", s::Signal)
-    writemime(io, m, s.value)
+@compat function Base.show(io::IO, m::MIME"text/plain", s::Signal)
+    Base.show(io, m, s.value)
```

Comment:
I think this needs to be `@compat show(io, m, s.value)`, since that will call `writemime` on 0.4